diff --git a/node_modules/metro-file-map/src/watchers/FallbackWatcher.js b/node_modules/metro-file-map/src/watchers/FallbackWatcher.js
index fe12544..fe23ed5 100644
--- a/node_modules/metro-file-map/src/watchers/FallbackWatcher.js
+++ b/node_modules/metro-file-map/src/watchers/FallbackWatcher.js
@@ -326,7 +326,9 @@ class FallbackWatcher extends _AbstractWatcher.AbstractWatcher {
 exports.default = FallbackWatcher;
 function isIgnorableFileError(error) {
   return (
-    error.code === "ENOENT" || (error.code === "EPERM" && platform === "win32")
+    error.code === "ENOENT" ||
+    (error.code === "EPERM" && platform === "win32") ||
+    error.code === "EACCES"
   );
 }
 function recReaddir(
